tool/tctl/common/resource_command.go

```diff
@@ -213,6 +214,19 @@ func (u *ResourceCommand) createTrustedCluster(client auth.ClientI, raw services
 	return nil
 }
 
+// createCertAuthority creates certificate authority
+func (u *ResourceCommand) createCertAuthority(client auth.ClientI, raw services.UnknownResource) error {
+	certAuthority, err := services.GetCertAuthorityMarshaler().UnmarshalCertAuthority(raw.Raw)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+	if err := client.UpsertCertAuthority(certAuthority); err != nil {
```

Comment:
Since this does a upsert and not create it's a pretty serious operation with which you can accidentally take down your entire cluster. We should probably prompt the user with something like before continuing.

```
Warning: Updating your certificate authorities can potentially put your cluster in a unrecoverable
state. Make sure you backup all Teleport state and export your existing certificate authorities
before you continue.

Are you sure you want to proceed?
```

Reply:
we don't have this flow anywhere, so I would let them do this on their own risk.